server/src/main/java/org/tctalent/server/util/html/StringSanitizer.java

```diff
@@ -56,4 +61,40 @@ public static String replaceLsepWithBr(String input) {
     return input.replace(LSEP, "<br>");
   }
 
+  /**
+   * Removes control characters from the input string.
+   * <p>
+   * This method removes ASCII control characters (characters with code points from 0x00-0x1F),
+   * except for common whitespace characters: newline (0x0A), carriage return (0x0D) and tab (0x09),
+   * which are preserved.
+   * <p>
+   * If control characters are found and removed, a debug log message is generated with the input
+   * string (truncated to 500 characters if it's longer).
+   *
+   * @param input the string to sanitize, can be null
+   * @return the sanitized string with control characters removed, or null if the input was null
+   */
+  @Nullable
+  static String replaceControlCharacters(@Nullable String input) {
```

Comment:
Can you rename this to removeControlCharacters(...) instead of replace, which is closer in meaning to its purpose

```suggestion
  static String removeControlCharacters(@Nullable String input) {
```

Reply:
Great point, @sadatmalik! Oversight on my part - I was originally replacing with a question mark, like John's similar code in the Opportunity class, but ultimately opted for this approach. Resubmitted with your suggestion.